include/ccf/endpoint.h

```diff
@@ -111,6 +111,9 @@ namespace ccf::endpoints
     /// Full URI path to endpoint, including method prefix
     URI full_uri_path;
 
+    /// URI path for the API documentation, that reflects the latest changes
+    URI api_uri_path;
```

Comment:
I'm not entirely clear on what "the latest changes" means in that context. Is this effectively the api uri path without the frontend/actor prefix?

Reply:
Yes, that's correct. But that bit is unnecessary (and unclear) in the comment. Will remove.